api/models/node.go

```diff
@@ -122,6 +122,19 @@ func (t IndexAutoCompleteTagValues) Trace(span opentracing.Span) {
 func (i IndexAutoCompleteTagValues) TraceDebug(span opentracing.Span) {
 }
 
+type IndexTagDelSeries struct {
+	OrgId int      `json:"orgId" binding:"Required"`
+	Paths []string `json:"path" form:"path"`
```

Comment:
interesting that is called "paths". afaik the term "path" is not commonly used yet. but i see it's the same on http://graphite.readthedocs.io/en/latest/tags.html
it looks like these values are used for the values for name tags, so why not just call this name tags ( @DanCech question for you ) :) anyway not worth holding back this PR for my nitpickery but it just caught my eye.

Reply:
i think the term name already has an implied meaning of `name` wihout any tags, because we also have a virtual tag called `name` which has that content.
So we need an alternative word to describe `name;<tags>...` and because `nameWithTags` path might be better